src/libs/actions/Policy/Policy.ts

```diff
@@ -165,6 +165,19 @@ Onyx.connect({
     },
 });
 
+let allReportActions: OnyxCollection<ReportActions>;
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.REPORT_ACTIONS,
+    waitForCollectionCallback: true,
+    callback: (actions) => {
+        if (!actions) {
```

Comment:
How can this be null/falsy?

Reply:
That is the way how we are getting reportActions through the whole application

Reply:
Hmmm you are right. I wonder why...

Reply:
But agree that your statement makes sense here, maybe to prevent any issues when undefined can come, not a Record for some reason